examples/test-cro1.yaml

```diff
@@ -19,5 +19,5 @@ spec:
                   env: canary
         jsonPatchOverrides:
           - op: add
-            path: /metadata/labels/new-label
-            value: "new-value"
\ No newline at end of file
+            path: /metadata/labels
+            value: {"new-label": "new-value"}

```

Comment:
hi ryan, this example is to add labels when the list is empty.

We need an example: add a new label when the list is not empty.
The original example is supposed to work :(